### DISCUSSION TITLE
trigger x stato non disponibile

### DISCUSSION THREAD

**Discussion by pifma** [indomus/forum#2935] [HUB: Home Assistant]
mi capita spesso che le automazioni che imposto nel caso che la batteria scende sotto il 30% non scattano perché il sensore di alcuni apparati wifi (shelly) diventa "non disponibile"; oltre al motivo che non mi è chiaro vorrei monitorare lo stato in modo di avvertirmi quando un sensore non è disponibile (mi basterebbe anche un controllo giornaliero).
![IMG_20240224_113229_168](https://github.com/indomus/forum/assets/130249205/b611b1d5-c94f-4dfa-a335-e69ad2b42e8b)

**Comment by snakuzzo** [comment]
Fai un'automazione con un trigger sullo stato dell'entità quando passa a `unavailable` e nella action il notify.

**Comment by pifma** [comment]
Ho fatto delle prove e dovrebbe funzionare, mi sorge però un altro problema che il dispositivo, essendo a batterie, potrebbe essere visto da ha come unavailable solo dopo settimane ( si può intravedere il comportamento anche dall'immagine ).
Esiste un modo che controlla le entità del dispositivo (es. Luminosità o temperatura) e se non manda più dati nell'arco di 24 h mi scatena un trigger?

**Reply by snakuzzo** [reply]
Puoi utilizzare `last_updated` o `last_changed` (che non è detto che coincidano) in base a quello che ti occorre...

Es.

```yaml
{{ states.sensor.porta_capannone.last_updated }}
```

oppure

```yaml
{{ states.sensor.porta_capannone.last_changed }}
```

**Reply by pifma** [reply]
Grazie
